zerver/models.py

```diff
@@ -550,6 +555,16 @@ class PreregistrationUser(models.Model):
 
     realm = models.ForeignKey(Realm, null=True) # type: Optional[Realm]
 
+class OutgoingWebhookBot(UserProfile):
+    base_url = models.TextField() # type: text_type
+    service_api_key = models.TextField() # type: text_type
```

Comment:
service_api_key should probably be a CharField(32) or something (with whatever length we're using for these API keys).